tests/env.py

```diff
@@ -10,10 +11,15 @@
 class Environment(object):
 
     def setup(self):
+        self.setup_collations()
         for i in range(0, 2):
             self.setup_translations(i)
         self.setup_x()
 
+    def setup_collations(self):
+        cursor = connection.cursor()
+        cursor.execute("CREATE COLLATION tr_tr (LOCALE = 'tr_TR.utf8');")
```

Comment:
Here again I'm worried that we're doing the same thing in (as far as I know) 2 different places. Do you have any ideas why it doesn't work in travis without this fixture?

Reply:
no, but after a lot of re/running travis what i could see was adding the collation and then dumping the collation table showed it to be present, but when you actually run the tests the collation was missing

This really was the only way i could find to get this to work in travis, and as it removes the other setup, its only done once

Reply:
...it also means its defo present in local test runs, so its actually a lot more reliable doing it this way